nsdb-common/src/main/scala/io/radicalbit/nsdb/common/statement/SQLStatement.scala

```diff
@@ -219,31 +221,48 @@ final case class LimitOperator(value: Int) extends NSDbSerializable
     new JsonSubTypes.Type(value = classOf[RelativeComparisonValue], name = "RelativeComparisonValue")
   ))
 sealed trait ComparisonValue {
+  def absoluteValue(currentTime: Long): NSDbType
   def value: NSDbType
 }
 
 object ComparisonValue {
-  def unapply(cv: ComparisonValue): Option[NSDbType] = Some(cv.value)
+
+  /**
+    * Extract the value of a [[ComparisonValue]]
+    * @return a tuple containing the value and a boolean that is true whether the comparison is relative.
+    */
+  def unapply(comparisonValue: ComparisonValue): Option[NSDbType] =
+    Some(comparisonValue.value)
+
 }
 
 /**
   * Class that represent an absolute comparison value
   * @param value the absolute value
   */
```

Comment:
I remember these things :)

Reply:
ehehe, in the next PR we'll add a brand new clause: `since [x] [time measure]` for temporal group by queries :)